voice/voice-ai/x/SpeechRecognition/SpeechRecognition.swift

```diff
@@ -373,6 +378,8 @@ class SpeechRecognition: NSObject, ObservableObject, SpeechRecognitionProtocol {
                         self.conversation.append(Message(role: "assistant", content: self.completeResponse.joined()))
                     }
                     transaction.finish()
+                    self.timeLogger?.finishAIResponse()
+                    self.timeLogger?.sendLog()
```

Comment:
shouldn't send log now because tts has not started

Reply:
also this doesn't necessarily happen - for example if it gets cancelled or there is an error, we would not get into this branch of execution